spec/bojangles_spec.rb

```diff
@@ -13,11 +13,14 @@
                 'Host' => 'bustracker.pvta.com',
                 'User-Agent' => 'Ruby'
               }).to_return(status: 200, body: '', headers: {})
-      allow(Pony).to receive(:deliver)
+      issue = double number: 123
+      client = double create_issue: issue, add_comment: nil
```

Comment:
Oh, I like what you did here. Instead of mocking.

Reply:
Well, it is a mock, just earlier up the chain.

Reply:
Okay. Well I think it's good.